data/divers/divers . ameublement.yaml

```diff
@@ -394,53 +411,84 @@ divers . ameublement . grand meuble . durée:
 divers . ameublement . grand meuble . nombre:
   titre: Nombre de grands meubles possédés
   question: Combien de grands meubles (non inclu ailleurs) possédez-vous ?
-  par défaut: 2
+  par défaut: 5
   période: année
 
 divers . ameublement . grand meuble:
   titre: Grand meuble
-  description: Par exemple un grand bureau, un meuble de cuisine
+  description: Meuble de cuisine, grand bureau
   icônes: 🛏️
   formule: empreinte amortie * nombre
   unité: kgCO2e
 
-divers . ameublement . préservation:
-  question: Quel effort faites-vous pour garder vos meubles le plus longtemps possible ?
-  description: |
-    Pour éviter de vous demander de renseigner l'âge précis de tous les biens d'ameublement que vous possédez, nous avons défini un âge moyen pour chacun des biens (généralement 10 ans).
+divers . ameublement . salon de jardin bois . empreinte amortie:
+  titre: Empreinte salon de jardin en bois amortie
+  formule: salon de jardin bois . empreinte / (durée * coefficient préservation)
+  unité: kgCO2e
 
-    > Nous cherchons ici à évaluer, non pas l'âge des meubles avant que ces derniers deviennent inutlisables, mais l'âge quand leurs propriétaires décident de les remplacer par du neuf.
-    A vous de vous placer par rapport à la moyenne.
-  par défaut: "'moyen'"
-  formule:
-    une possibilité:
-      choix obligatoire: oui
-      possibilités:
-        - achat neuf
-        - faible
-        - moyen
-        - maximum
+divers . ameublement . salon de jardin bois . empreinte:
+  titre: Empreinte salon de jardin en bois
+  formule: 69.2
```

Comment:
Je n'ai actuellement plus accès à la base carbone (j'attends mon mot de passe pour le nouveau site :disappointed:), mais je pense qu'il faut préciser ici si l'empreinte inclut la fixation du CO2 par le bois. 

Dans mes souvenis, la base carbone présente deux empreintes, l'une avec CO2 capté et l'autre non. 

CF cette issue https://github.com/datagir/nosgestesclimat/issues/1490. J'avoue avoir du mal à comprendre pourquoi on ne compte pas ces puits, mais de toutes façon il faut qu'on l'explicite.

Reply:
@laem 

> Dans mes souvenirs, la base carbone présente deux empreintes, l'une avec CO2 capté et l'autre non.

C'est quasi ça. La Base Carbone comptabilise le CO2 biogéniques (qu'il soit capté ou brulé) à part du CO2 fossile.

C'est une convention de calcul des facteurs d'émissions liées à plusieurs raisons : 
-  dans le cas de CO2 biogénique capté, il est difficile de savoir l'origine du bois et il est donc choisi de ne pas considérer que le bois est issu de forêts gérées durablement par principe. Cette info est donc présentée "à côté". Même si, pour des produits bois français, il y a de grandes chances que le bois soit issu de forêts gérées durablement.
- la dynamique très longue de la régénération des stocks (influencés par la durée de pousse considérée, elle même liée à l'intensité des coupes mais aussi par les aléas météo par exemple ou encore par la pratique sylvicole qui peut influencer les dynamiques de captage et donc les stocks) qui fait ,qu'en réalité, l'impact positif de bois même issu de forêts gérées durablement peut être contestable

Je propose de garder cette approche conservatrice et de ne pas considérer, dans une première version du modèle, de séquestration par le produit bois. 
On pourrait ensuite réfléchir à une question/option/action (a voir comment on tourne ça) mettant en avant l'intérêt des produits bois issus de forêts "durables" car cela permet de séquestrer du CO2, non ?